index.html

```diff
@@ -70,6 +70,215 @@ <h1>
+            <div class="card-button">
+              <button type="button" class="button">
+                See Project
+              </button>
+            </div>
+          </div>
+        </div>
+        <div class="card-container">
+          <div class="card-image"></div>
+          <div class="card-body">
+            <div class="card-title">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+            </div>
+            <div class="card-content">
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>html</li>
+              </ul>
+            </div>
+            <div class="card-button">
+              <button type="button" class="button">
+                See Project
+              </button>
+            </div>
+          </div>
+        </div>
+        <div class="card-container">
+          <div class="card-image"></div>
+          <div class="card-body">
+            <div class="card-title">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+            </div>
+            <div class="card-content">
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>html</li>
+              </ul>
+            </div>
+            <div class="card-button">
+              <button type="button" class="button">
+                See Project
+              </button>
+            </div>
+          </div>
+        </div>
+        <div class="card-container">
+          <div class="card-image"></div>
+          <div class="card-body">
+            <div class="card-title">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+            </div>
+            <div class="card-content">
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>html</li>
+              </ul>
+            </div>
+            <div class="card-button">
+              <button type="button" class="button">
+                See Project
+              </button>
+            </div>
+          </div>
+        </div>
+        <div class="card-container">
+          <div class="card-image"></div>
+          <div class="card-body">
+            <div class="card-title">
+              <h2>Multi-Post Stories Gain+Glory</h2>
+            </div>
+            <div class="card-content">
+              <ul>
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Javascript</li>
+                <li>html</li>
+              </ul>
+            </div>
+            <div class="card-button">
+              <button type="button" class="button">
+                See Project
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section id="about-myself">
+      <img src="images/frameup.png" alt="" class="img-up">
```

Comment:
- [ ] This is not an image this is your background, so please make this your **about-myself** section background through CSS. :+1: